src/ucs/vfs/sock/vfs_sock.h

```diff
@@ -7,8 +7,83 @@
 #ifndef UCS_VFS_SOCK_H_
 #define UCS_VFS_SOCK_H_
 
+#include <sys/types.h>
+#include <sys/un.h>
+#include <stdint.h>
+
 /* This header file defines socket operations for communicating between UCS
  * library and VFS daemon */
 
+/**
+ * VFS socket message type
+ */
+typedef enum {
+    UCS_VFS_SOCK_ACTION_STOP, /* daemon is asked to stop */
+    UCS_VFS_SOCK_ACTION_MOUNT, /* daemon is asked to mount a file system */
+    UCS_VFS_SOCK_ACTION_MOUNT_REPLY, /* daemon sends back FUSE file descriptor */
+    UCS_VFS_SOCK_ACTION_LAST,
+    UCS_VFS_SOCK_ACTION_INVALID = -1
```

Comment:
can remove `UCS_VFS_SOCK_ACTION_INVALID` and use `UCS_VFS_SOCK_ACTION_LAST` instead of it
wdyt?